msteams-platform/tutorials/get-started-yeoman.md

```diff
@@ -129,4 +138,39 @@ Choose your tab and follow the instructions to add it. Notice that you have a cu
 
 ![running tab in teams](~/assets/yeoman-images/teams-first-app-6.png)
 
-**Congrats! You built and deployed your first Microsoft Teams app**
+## Upgrade Microsoft Teams
+You can also upgrade your current Microsoft Teams version to the latest version using the Microsoft Teams Yeoman generator.
+
+**To upgrade Microsoft Teams**
```

Comment:
@KirtiPereira I do not think this is needed when the heading already conveys this.

Reply:
@heath-hamilton - Will keep this for now. Per MSTP convention, it is good to have a procedural heading. 
Thanks.